R/cnd.R

```diff
@@ -435,6 +435,87 @@ validate_signal_args <- function(msg, type, call, env = parent.frame()) {
   }
 }
 
+#' Muffle a condition
+#'
+#' Unlike [exiting()] handlers, [calling()] handlers must be explicit
+#' that they have handled a condition to stop it from propagating to
+#' other handlers. Use `cnd_muffle()` within a calling handler (or as
+#' a calling handler, see examples) to prevent any other handlers from
+#' being called for that condition.
+#'
+#'
+#' @section Mufflable conditions:
+#'
+#' Not all conditions are signalled with a muffling restart.
```

Comment:
I think this needs a brief lead-in like: Most conditions signalled by base are muffable, although the name of the restart varies. `cnd_muffle()` will automatically call the correct restart for ...